src/Dropzone.vue

```diff
@@ -154,7 +154,7 @@ export default {
                 this.$emit('uploading', false);
                 this.$emit('file-upload-error', {
                     file: file,
-                    response: response,
+                    errorMessage: errorMessage,
```

Comment:
We have changed key names here, is this going to introduce any breaking changes for code expecting the old `response` key?

Reply:
Thats a good point @BrockReece, i guess it makes sense to keep the key name the same. Thanks